CefSharp.OffScreen.Example/CefSharp.OffScreen.Example.netcore.csproj

```diff
@@ -17,22 +17,28 @@
         <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
         <ApplicationManifest>app.manifest</ApplicationManifest>
         <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
-        <Platforms>x86;x64</Platforms>
+        <Platforms>x86;x64;arm64</Platforms>
+        <!-- Explicitly set the PlatformTarget for arm64, see: https://github.com/dotnet/sdk/issues/15434 -->
+        <PlatformTarget Condition="'$(Platform)' == 'arm64'">arm64</PlatformTarget>
         <StartupObject>CefSharp.OffScreen.Example.Program</StartupObject>
         <!--
         Run on a newer version of .Net if the specified version (.Net Core 3.1) is not installed
         https://github.com/dotnet/docs/issues/12237
         -->
         <RollForward>Major</RollForward>
+        <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
```

Comment:
Is the `<RuntimeIdentifiers/>` (plural) element required here? I was under the impression it was one or other.

Reply:
It seems to be required here for `build.netcore.ps1` to work (using `MSBuild` from VS, not `dotnet build`). Otherwise, MSBuild complains about the runtime identifier used (e.g. `win-x64`) not being present in `obj\project.assets.json`, and suggests to add it to `<RuntimeIdentifiers>`.

I think it is for the `restore` operation which needs to know what runtime packages to restore before building the project (though I'm not sure why it works without specifying `<RuntimeIdentifiers>` when building directly in Visual Studio).